src/commandsAndMenu.tsx

```diff
@@ -254,8 +255,8 @@ export function addCommands(
 
   /** Command to add a remote Git repository */
   commands.addCommand(CommandIDs.gitAddRemote, {
```

Comment:
Let's refactor this a bit. Would you mind renaming the `gitAddRemote` command to be `gitManageRemote`.

We then should remove the ability to pass args to the `execute` method. Then in the dialog we should do the addition without closing the dialog. So the dialog is respecting its contract of management not of single action.

Reply:
Renamed the command to gitManageRemote and disabled arguments for the command. 
To-do:
Implement adding a new remote without closing the dialog